### ISSUE TITLE
lib64 dir

### ISSUE THREAD

**Issue by bioinfornatics** [dlang/dub#93]
on fedora and other linux distro rpm based when a lib is build on 64, lib go to /usr/lib64.
So how dub manage this?

And another point when i do a package i do not want that dub will download dep and build it this should to be done by the packager in another package. So how to manage this ?

**Comment by dhasenan** [comment]
tldr: should probably close this.

If you're packaging a library for a specific OS, you probably need to do more than dub can reasonably do. If you just need it to be in an appropriate location, that's OS-specific, distro-specific. FreeBSD64 uses `/usr/lib`, apparently some stuff uses `/usr/lib64`, and these days Linuxes tend to use `/usr/lib/x86_64-linux-gnu`. Windows doesn't recommend installing libraries system-wide. I'm not sure, but OSX seems to be using `/usr/lib` directly.

This all changes over time. And does Arch Linux use the same path as Ubuntu? What about OpenBSD? Or Solaris? What about 32-bit libs on all of the above?

So I don't think this is really appropriate for dub to handle.

@chmike That's unrelated. It looks like more modern Ubuntus make those symlinks by default, so it's moderately safe to call it a packaging issue.

**Comment by wilzbach** [comment]
Well making dub play nicely with package managers is an outstanding issue e.g. #838, #839, thus are many other open issues for the bits that dub could do well
